exotic/tests/test_utils.py

```diff
@@ -0,0 +1,58 @@
+from exotic.utils import round_to_2
```

Comment:
why the new tests folder?

Reply:
It's a way to organize tests that keeps the tests 1) close to the code and 2) the implementation code uncluttered.

Documentation for python-conventional ways to organize tests is here: https://docs.pytest.org/en/6.2.x/goodpractices.html#choosing-a-test-layout-import-rules I chose the approach that made the most sense.

Reply:
There are quite a few open source, popular, python repositories that use this paradigm of testing. Check out [qlib](https://github.com/microsoft/qlib) and it's [top level test directory](https://github.com/microsoft/qlib/tree/main/tests), [sklearn](https://github.com/scikit-learn/scikit-learn/tree/main/sklearn) and [numpy](https://github.com/numpy/numpy/tree/main/numpy) to see how they've implemented their test suites. I used these to guide this and my other testing branches.

I'm happy to chat through other paradigms if you would like. I'm also happy to change paradigms in the future (and backport these) as we get more testing code implemented and a different paradigm reveals itself as advantageous.